src/common/components/elements/ModelViewer.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import '@google/model-viewer';
+
+const ModelViewer = (props: any) => (
+  // @ts-ignore
+  <model-viewer {...props} style={{ width: 800, height: 400 }}></model-viewer>
```

Comment:
This is going to cause mobile viewing issues. Ideal is having model view take the width of the container and the height is the same as the width (square). Use padding trick leveraged by image previews to achieve.